### DISCUSSION TITLE
How to get real siteUrl (gatsby) on gatsby feed rss plugin with Wordpress backend?

### DISCUSSION THREAD

**Discussion by mrenzo20** [gatsbyjs/gatsby#37638] [Help]
I have a wordpress + gatsby developemnt.
I configure official feed rss gatsby plugin but on options query, try to get siteUrl and i just need gastby url and not wordpress url. How i can get gastby public url?
Thanks.

**Comment by gh640** [comment]
You can get the `siteUrl` in `site.siteMetadata`. The following page has an example. Does this help?

https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-an-rss-feed/
